src/main/clm_initializeMod.F90

```diff
@@ -293,8 +293,12 @@ subroutine initialize2( )
     use controlMod            , only : NLFilename
     use clm_instMod           , only : clm_fates
     use BalanceCheckMod       , only : BalanceCheckInit
+    use FATESFireNoDataMod, only: fates_fire_no_data_type
+    use FATESFireDataMod, only: fates_fire_data_type
     !
     ! !ARGUMENTS    
+    type(fates_fire_data_type) :: fates_fire_data_inst
```

Comment:
Just use the generic type rather than the specific one

Reply:
Same [comment](https://github.com/ESCOMP/CTSM/pull/991#discussion_r415423650).